### PR TITLE
Correctly handle sending unknown toilet code entries to Ona.

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: src/app.js [hunk 1]
```diff
@@ -283,6 +283,34 @@ go.app = function() {
...
+                    data.toilet.lat + offsets.lat,
+                    data.toilet.lon + offsets.lon,
+                ].join(' ');
+            }
+            submission = _.defaults(submission, {
+                toilet_code: data.query,
+                toilet_section: "None",
+                toilet_cluster: "None",
+                issue: data.issue,
+                toilet_location: "None",
```
Comment (rudigiesler) [inline src/app.js]: Doing some testing, you get a 500 error with `toilet_location: "None"`. The submission goes through with `0 0`, and doesn't show up on the map, although I'm not sure why.

**Comment by rudigiesler** [comment]
Just one comment about the defaults

**Comment by hodgestar** [comment]
What happens if we leave toilet_location out entirely?

**Comment by rudigiesler** [comment]
Yes that works. Leaving it out doesn't display on the map, and results in a successful submission.

**Comment by hodgestar** [comment]
Ready for re-review.

**Comment by rudigiesler** [comment]
:+1:

**Pull request closed without merging**